metaflow/plugins/aws/step_functions/step_functions_cli.py

```diff
@@ -97,6 +98,12 @@ def step_functions(obj, name=None):
     "with the given tag. You can specify this option multiple "
     "times to attach multiple tags.",
 )
+@click.option(
+    "--aws-batch-tags",
```

Comment:
If we are to keep this, the flag itself should be singular in my opinion as it is used to set one tag at a time
```
--aws-batch-tag some=tag --aws-batch-tag another=tag
```